.github/workflows/build-release.yml

```diff
@@ -26,6 +26,8 @@ jobs:
         include:
           - os: macos-12
             platform: 'darwin-x64'
+          - os: macos-12
```

Comment:
I think you'll probably need to add this platform back into the list here:

https://github.com/animoto/sharp-libvips/blob/883eb627be4b8a63248f866e2bba9105d4d17d31/build.sh#L104

Reply:
Oh wait, that's just the linux ones, so never mind...